treeMap.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <div>
+        <svg
+            id="treeMapViz"
+            :viewBox='[0, 0, 100, 100]'
+            class=".viewBoxStyle" >
+            <g 
+                v-for="(leaf, index) in this.treeMapLeaves"
+                    :key="index"
+                    :transform="'translate(' + leaf.x0 + ',' + leaf.y0 +')'"
+                    >
+                <rect
+                    :id="index +'leaf'"
+                    :width="rectWidth(leaf)"
+                    :height="rectHeight(leaf)"
+                    class=".rectStyle"
+                >
+                </rect>
+            </g>
+        </svg>
+    </div>    
+</template>
+<script >
+import * as d3 from 'd3'
+export default {
+    data(){
+        return {
+            width: {
+                type: Number,
+                default: 954
+            },
+            height: {
+                type: Number,
+                default: 954
+            }
+        }
+    },
+    props:{
+        users: {
+            type: Array,
+            default: function() {
+                return []
+            }
+        }
+    },
+    computed:{
+        treeMapData: function(){
+            let root = this.treeMapFunc(this.users)

```

Comment:
Why are you calling it on `this.users`? (you have to first transform users into another data type, a data type that fits the requirements of `d3.treemap` function)